bin/desi_fit_guide_star_coordinates

```diff
@@ -24,11 +24,22 @@ log   = get_logger()
 
 fm = FieldModel()
 
-header  = pyfits.open(args.fits_header)[1].header
+header  = fitsio.read_header(args.fits_header,0)
+if not "TARGTRA" in header :
+    log.warning("no TARGTRA in header of HDU 0, try HDU 1")
+    header  = fitsio.read_header(args.fits_header,1)
+    if not "TARGTRA" in header :
+        log.error("no TARGTRA in headers of HDU 0 or 1 of file {}".format(args.fits_header))
+        sys.exit(12)
+
+        
 fm.ra  = header["TARGTRA"]
 fm.dec = header["TARGTDEC"]
 fm.expid = header["EXPID"]
-fm.hexrot_deg = float(header["FOCUS"].split(",")[5])/3600.
+fm.hexrot_deg = float(header["FOCUS"][5])/3600.
```

Comment:
checking: is this change due to switching from astropy.io.fits to fitsio, or is this another gfa vs. guide header and/or evolving header formats that we need to support both splitting a string by "," and getting an element from an array?

Reply:
astropy -> fitsio